extensions/amp-user-notification/0.1/amp-user-notification.js

```diff
@@ -308,14 +333,17 @@ export class AmpUserNotification extends AMP.BaseElement {
   /**
    * Hides the current user notification and invokes the `dialogResolve_`
    * method. Removes the `.amp-active` class from the element.
+   *
+   * @param {boolean=} opt_forceNoPersist If true, dismissal won't be persisted
+   * regardless of 'data-persist-dismissal''s value
    */
-  dismiss() {
+  dismiss(opt_forceNoPersist = false) {
```

Comment:
wow good catch Justin. Only issue was `registerAction` ( which actually was caught after I enabled type checking ). 

Anyway now I am a bit worried about this. Should we start preferring explicit `() => func()` over `bind`?